Bio/PDB/Vector.py

```diff
@@ -10,194 +10,8 @@
 import numpy
 
 
-def m2rotaxis(m):
```

Comment:
Apologies, I thought they were testing methods since they were outside the Class definition.

I'll sort them out this weekend.